res/controllers/Denon-MC7000.midi.xml

```diff
@@ -0,0 +1,4054 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.2.3" schemaVersion="1">
+    <info>
+    	<name>Denon MC7000 beta for MIXXX 2.2.x</name>
```

Comment:
Can you change the name so that it matches all other controller mappings? It doesn't look good in the controller mapping combobox and is confusing because this mapping also works with 2.3.

```suggestion
    	<name>Denon MC7000</name>
```